mod/node-api/handlers/beacon/types/response.go

```diff
@@ -57,22 +50,75 @@ type RootData struct {
 	Root common.Root `json:"root"`
 }
 
-type ValidatorData[ValidatorT any] struct {
+type ValidatorBalanceData struct {
+	Index   uint64 `json:"index"`
+	Balance uint64 `json:"balance"`
+}
+
+func (vbd ValidatorBalanceData) MarshalJSON() ([]byte, error) {
+	return json.Marshal(struct {
+		Index   string `json:"index"`
+		Balance string `json:"balance"`
+	}{
+		Index:   strconv.FormatUint(vbd.Index, 10),
+		Balance: strconv.FormatUint(vbd.Balance, 10),
+	})
+}
+
+type ValidatorData[ValidatorT Validator] struct {
```

Comment:
do we have a standardization for how we want all thes etypes to be, they feel really all over the place

Reply:
As we are following Beacon API spec, anything that is labelled as <Entity>Data will be part of "data" in the API response.